install/disconnected_install.adoc

```diff
@@ -176,6 +176,14 @@ do
 done
 ----
 
+[NOTE]
+====
+If you are syncing images on each of the hosts, rather than storing them in a
+local repository to pull from, you must disable the Docker image check or
+Skopeo will attempt to pull images even though they are locally cached and
```

Comment:
What's Skopeo? Is it important? (I don't see that term anywhere else in the docs.)
How do I disable the check?
I think this info needs to be a step or prereq in the appropriate section. (Does it go in the next section instead of this one?)

Reply:
It's an Atomic utility, and honestly I'm on the fence with this one. It was requested as a note here specifically. It's an edge case, so it might not be necessary to document at all.

Reply:
Interesting! If we don't have this flow described somewhere, +1 to not documenting the edge case.